Content.Server/Weapon/Melee/ZombieTransfer/ZombieTransferSystem.cs

```diff
@@ -37,7 +37,7 @@ private void OnMeleeHit(EntityUid uid, ZombieTransferComponent component, MeleeH
                 if (args.User == entity)
                     continue;
 
-                if (!HasComp<MobStateComponent>(entity))
+                if (!HasComp<MobStateComponent>(entity) || HasComp<DroneComponent>(entity))
```

Comment:
Should this just check whether the entity has the `DiseaseCarrierComponent` instead?

Reply:
This just prevents drones from being zombified. I'm aware this implementation is scuffed and i'm going to go through and fix it in another pr. This was just a small thing.

Reply:
I mean that wouldn't checking for DiseaseCarrierComponent also prevent it and other things that shouldn't be zombified from being zombified or are there mobs without DiseasCarrierComponent that can be zombified?

Reply:
The system actually allows for all mobs to be converted, even if they don't have diseaseCarrierComponent. This is really just in place to allow things like pets and monkeys to be converted without having them also pick up diseases (because no one wants to see ian get cancer). It just doesn't give them infection and instead does the regular conversion upon kill.